test/parsed_julia_type.jl

```diff
@@ -0,0 +1,19 @@
+using StaticArrays
+
+include("../src/parse_type.jl")
+@test eval(Meta.parse(get_julia_type("double"))) == Float64
+@test eval(Meta.parse(get_julia_type("bool"))) == Bool
+@test eval(Meta.parse(get_julia_type("int"))) == Int64
+@test eval(Meta.parse(get_julia_type("float"))) == Float32
+@test eval(Meta.parse(get_julia_type("int8_t"))) == Int8
+@test eval(Meta.parse(get_julia_type("uint8_t"))) == UInt8
+@test eval(Meta.parse(get_julia_type("int16_t"))) == Int16
+@test eval(Meta.parse(get_julia_type("uint16_t"))) == UInt16
+@test eval(Meta.parse(get_julia_type("int32_t"))) == Int32
+@test eval(Meta.parse(get_julia_type("uint32_t"))) == UInt32
+@test eval(Meta.parse(get_julia_type("int64_t"))) == Int64
+@test eval(Meta.parse(get_julia_type("uint64_t"))) == UInt64
```

Comment:
```suggestion
@test eval(Meta.parse(get_julia_type("uint64_t"))) == UInt64
@test eval(Meta.parse(get_julia_type("Vector3f"))) == Vector3f
@test eval(Meta.parse(get_julia_type("SomeComponent"))) == SomeComponent
```

Just a quick note on these tests: Here you actually check that what Julia parses from the output of `get_julia_type` is the same as the expected type. For the podio case I am not entirely sure that will be possible for all the use cases, as some of the types that we try to compare against will only exist after we have generated the code. One way out would be to just compare the string output of `get_juliat_type`, i.e.
```julia
@test get_julia_type("Component") == "Component"
@test get_julia_type("std::array<int32_t, 3>") == "MVector{3, Int32}"
```

Reply:
But also keep in mind that this functionality has to be moved to podio and python in any case, so don't spend too much time on the tests here, as they will also need to be moved to podio (and python).